app/api/dao/mentorship_relation.py

```diff
@@ -365,3 +365,27 @@ def list_pending_mentorship_relations(user_id):
                 pending_requests += [relation]
 
         return pending_requests, 200
+
+    @staticmethod
+    @email_verification_required
+    def count_pending_mentorship_relations(user_id):
+        """Counts all the pending mentorship requests of the user.
+
+        Args:
+            user_id: ID of the user counting all his pending mentorship requests.
+
+        Returns:
+            message: A message containing the number of pending mentorship requests of the user along with the success status code,
+                     or failure status code if otherwise.
```

Comment:
`success status code, or failure status code` - does it really return any status here?